components/images/fullpage-hero.css

```diff
@@ -22,7 +22,7 @@
 .ampstart-image-fullpage-hero .ampstart-image-credit {
   box-decoration-break: clone;
   background: var(--color-primary);
-  padding: 0 var(--space-2) 0.2rem;
```

Comment:
@wassgha the "Read more" looks fine, so if we want the same effect for the Title Text (which we do) than the black needs to extend above the text, below the text, to the left, and right, the same amount.  Right now its 18px from the top / bottom, and 8px from the right / left of the text. Before was actually closer : 11px from the top /bottom, and 18px from the right / left. I recommend 16px top / bottom and 16px left / right padding round the text. It's divisible by 8px, which is the system our type size is built on, so it should look proportionally the best.  Does that make sense?

//cc @camelburrito also this PR name should be more descriptive...? Is this referencing an issue?